pyenv-win/install-pyenv-win.ps1

```diff
@@ -118,7 +118,7 @@ Function Main() {
 
     (New-Object System.Net.WebClient).DownloadFile("https://github.com/pyenv-win/pyenv-win/archive/master.zip", $DownloadPath)
 
-    Start-Process -FilePath "powershell.exe" -ArgumentList @(
+    Start-Process -FilePath "pwsh.exe" -ArgumentList @(
```

Comment:
pwsh doesn't comes default with Windows 11 Pro, if we change powershell to pwsh this might effect other too.

Reply:
sorry for late, I see, so it's no need to be accepted. I don't know this, thanks.